lib/src/query_component.dart

```diff
@@ -53,6 +53,13 @@ class QueryComponent {
   bool showAlert = false;
   String alertMessage;
 
+  // The earliest startDate that reporting API can take.
+  DateTime reportStartDate = DateTime(
+      DateTime.now().year - 2, DateTime.now().month, DateTime.now().day);
+
```

Comment:
Seems like the earliest start date for reporting api is Now() minus 2 years.  I will double check this with Caleb.

Reply:
ok please leave issue with todo
what happens when you submit a time earlier than that?
a const may be better here.

Reply:
Caleb replied that the default look up window is 2 years. If a time earlier than that is provided, the reporting api will respond with aN error message in json.

Reply:
Hmm, couldn't really use const here since DateTime.now() is not const. I guess I will use static final then.